tools/sst_dump_test.cc

```diff
@@ -110,7 +110,7 @@ class SSTDumpToolTest : public testing::Test {
   template<std::size_t N>
   void PopulateCommandArgs(const std::string& file_path, const char* command,
     char* (&usage)[N]) const {
-    for (int i = 0; i < N; ++i) {
+    for (int i = 0; i < static_cast<int>(N); ++i) {
```

Comment:
why not for (size_t i = 0; i < N; ++i)?

Reply:
Don't have a strong opinion on this and I think either should work.
On 32-bit platforms (rare in modern enterprise servers), two registers will be needed to perform 64-bit arithmetic.